File: examples-sdk/zig/506-villog-structured-events/main.zig
// 506-villog-structured-events — Zig SDK equivalent
// Compile: vil compile --from zig --input 506-villog-structured-events/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}

File: examples-sdk/java/506-villog-structured-events/Main.java
// 506-villog-structured-events — Java SDK equivalent
// Compile: vil compile --from java --input 506-villog-structured-events/Main.java --release

import dev.vil.*;

public class Main {
    public static void main(String[] args) {
        VilServer server = new VilServer("app", 8080);
        server.compile();
    }
}
